recipes/libcufile/meta.yaml

```
@@ -0,0 +1,118 @@
+{% set name = "libcufile" %}
+{% set version = "1.5.0.59" %}
+{% set platform = "linux-x86_64" %}  # [linux64]
+{% set platform = "linux-sbsa" %}  # [aarch64]
+{% set extension = "tar.xz" %}  # [not win]
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/{{ name }}/{{ platform }}/{{ name }}-{{ platform }}-{{ version }}-archive.{{ extension }}
+  sha256: 222c49b29afb544c69542c4e658b0a1bd01fd2306f2136ad6990ccfff482a097  # [linux64]
+  sha256: 4e8d2b525322be9ab6ae574bd1d0325a4bf9c6c3bbdaa1611b5b969b7b34b0c4  # [aarch64]
+  patches:
+    - Makefile.patch
+
+build:
+  number: 0
+  skip: true  # [osx]
+  skip: true  # [ppc64le]
+  skip: true  # [aarch64]
+  skip: true  # [win]
+
+outputs:
+  - name: libcufile
+    build:
+      missing_dso_whitelist:
+        - "*/libcuda.so.*"                     # [linux]
+    run_exports:
+      - {{ pin_subpackage("libcufile", max_pin="x") }}
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+      host:                                    # [linux]
+        - sysroot_{{ target_platform }} 2.17   # [linux]
+        - __glibc >=2.17                       # [linux]
+      run:                                     # [linux]
+        - sysroot_{{ target_platform }} 2.17   # [linux]
+        - __glibc >=2.17                       # [linux]
+    files:
+      - lib/libcufile*.so.*
+      - etc/cufile.json
+      - man
+    test:
+      commands:
+        - test -f $PREFIX/lib/libcufile.so.*
+        - test -f $PREFIX/lib/libcufile_rdma.so.*
+        - test -f $PREFIX/etc/cufile.json
+
+  - name: libcufile-dev
+    run_exports:
+      - {{ pin_subpackage("libcufile", max_pin="x") }}
+    requirements:
+      run:
+        - libcufile >={{ version }}
+      run_contrained:
+        - libcufile-static >={{ version }}
```

Note there is a typo in `run_constrained`.
```suggestion
    build:
      run_exports:
        - {{ pin_subpackage("libcufile", max_pin="x") }}
    requirements:
      host:
        - cuda-version {{ cuda_version }}
      run:
        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
        - libcufile >={{ version }}
      run_constrained:
        - libcufile-static >={{ version }}
```